plugin/trino-iceberg/src/test/java/io/trino/plugin/iceberg/TestIcebergMigrateProcedure.java

```diff
@@ -224,20 +226,117 @@ public void testMigrateUnsupportedColumnType()
     {
         String tableName = "test_migrate_unsupported_column_type_" + randomNameSuffix();
         String hiveTableName = "hive.tpch." + tableName;
+
+        assertUpdate("CREATE TABLE " + hiveTableName + " AS SELECT tinyint '1' x", 1);
+        assertQuery("SELECT * FROM " + hiveTableName, "SELECT 1");
+
+        assertQueryFails("CALL iceberg.system.migrate('tpch', '%s')".formatted(tableName), "Type not supported for Iceberg: tinyint");
+
+        assertQuery("SELECT * FROM " + hiveTableName, "SELECT 1");
+
+        assertUpdate("DROP TABLE " + hiveTableName);
+    }
+
+    @DataProvider
+    public static Object[][] timestampPrecisionsAndValuesOrc()
+    {
+        return new Object[][] {
+                {HiveTimestampPrecision.MILLISECONDS, "2021-01-01 06:12:12.123", "2021-01-01 06:12:12.123"},
+                {HiveTimestampPrecision.MICROSECONDS, "2021-01-01 06:12:12.123456", "2021-01-01 06:12:12.123456"},
+                {HiveTimestampPrecision.NANOSECONDS, "2021-01-01 06:12:12.123456789", "2021-01-01 06:12:12.123457"} // Truncated as Parquet timestamp precision is 6
```

Comment:
method name suggests this is in context of ORC, the comment says Parquet. please explain

btw i believe that both Parquet and ORC support nano precision

Reply:
Fixed, It's supposed to be "Iceberg".